test/modules/ModuleLib/TestResults/EdgeHubRestartMessageResult.cs

```diff
@@ -27,7 +27,7 @@ public class EdgeHubRestartMessageResult : MessageTestResult
             this.MessageCompletedStatusCode = messageCompletedStatusCode;
         }
 
-        DateTime EdgeHubRestartedTime { get; set; }
+        public DateTime EdgeHubRestartedTime { get; set; }
 
         public DateTime MessageCompletedTime { get; set; }
 
```

Comment:
Why do you need "GetMessageTestResult()"?

Reply:
It's used to extract the `batchId;seqNum;trackId`